quick-start-with-htap.md

```diff
@@ -0,0 +1,216 @@
+- Data isolation of HTAP: TiKV and TiFlash can be deployed on different machines as needed to solve the problem of HTAP resource isolation.
+- MPP computing engine: [MPP](/tiflash/use-tiflash.md#control-whether-to-select-the-mpp-mode) is a distributed computing framework provided by the TiFlash engine since TiDB 5.0, which allows data exchange between nodes and provides high-performance, high-throughput SQL algorithms. If the MPP mode is enabled, the run time of the analytic queries can be significantly reduced.
+
+## Steps
+
+This document take a popular [TPC-H](http://www.tpc.org/tpch/) dataset as an example to guide you to experience the convenience and high performance of TiDB HTAP by trying to query.
+
+### Step 1: Prerequisites for deployment
+
+Before using TiDB HTAP, follow the steps in the [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md) to deploy a local test environment.
+
+In [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md):
+
+- You are recommended to run `tiup playground` to start a TiDB cluster of the latest version. When you run the following command, 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance are deloyed automatically:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup playground
+    ```
+
+- If you want to specify the TiDB version and the number of the instances of each component, run the command that specifies the number of the instances of TiFlash as following:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup playground v5.1.0 --db 2 --pd 3 --kv 3 --tiflash 1 --monitor
+    ```
+
+> **Note:**
+>
+> `tiup playground` command is ONLY for quick start, NOT for production.
+
+### Step 2: Prerequisites for data
+
+By following these steps, you can create the [TPC-H](http://www.tpc.org/tpch/) dataset for using TiDB HTAP. If you are interested in TPC-H, see [General Implementation Guidelines](http://tpc.org/tpc_documents_current_versions/pdf/tpc-h_v3.0.0.pdf).
+
+> **Note:**
+>
+> If you want to use your existing data for analytic queries, you can [migrate your data to TiDB](/migration-overview.md). If you want to design and create your own data, you can create it by executing SQL statements or using related tools.
+
+1. Install the tool that creates data by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup install bench
+    ```
+
+2. Create the data by by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup bench tpch --sf=1 prepare
+    ```
+
+    If the output of this command shows `Finished`, it indicates that the data is created.
+
+3. Execute the following SQL statement to see the created data:
+
+    {{< copyable "sql" >}}
+
+    ```sql
```

Comment:
Would be good to use `FORMAT_BYTES` here.
```
5.7.25-TiDB-v5.2.0-alpha-710-gbe4cdb28c 127.0.0.1:4000   test  SQL  SELECT FORMAT_BYTES(10000000000);
+---------------------------+
| FORMAT_BYTES(10000000000) |
+---------------------------+
| 9.31 GiB                  |
+---------------------------+
1 row in set (0.0006 sec)
```

Maybe format like this:
```
SELECT
  CONCAT(table_schema,'.',table_name) AS 'Table Name',
  table_rows AS 'Number of Rows',
  FORMAT_BYTES(data_length) AS 'Data Size',
  FORMAT_BYTES(index_length) AS 'Index Size',
  FORMAT_BYTES(data_length+index_length) AS'Total'
FROM
  information_schema.TABLES
WHERE
  table_schema='test';
```

Reply:
thanks for your suggestion